src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <label for="input-big" class="main__label main__label--big">
+        <img
+          src=".//images/Search.svg"
```

Comment:
The image source path is incorrect. It should not have two slashes after the dot.